llvm/test/CodeGen/AArch64/arm64-st1.ll

```diff
@@ -1,530 +1,550 @@
-; RUN: llc < %s -mtriple=arm64-eabi -aarch64-neon-syntax=apple -verify-machineinstrs | FileCheck %s
-; RUN: llc < %s -global-isel -global-isel-abort=1 -mtriple=arm64-eabi -aarch64-neon-syntax=apple -verify-machineinstrs | FileCheck %s
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --filter "(st|mov|add|zip)" --version 5
```

Comment:
It's dangerous to opt into specific instructions.  Given the CHECK lines are now auto-generated I'd just match the whole output. Alternatively, If there's some specific noise you want to eliminated then filter that specifically.  If I'm guessing I think for this you probably want `--filter-out kill`?

Reply:
I've updated this to use an equivalent `--filter-out` :+1: